lib/helpers.rb

```diff
@@ -34,8 +34,10 @@ def error_and_back(error)
   def set_client
     begin
       if settings.development?
+        Octokit.auto_paginate = true
         client = Octokit::Client.new(:netrc => true)
       else
+        Octokit.auto_paginate = true
```

Comment:
Okay, this lines are duplicated now, can you put it [here](https://github.com/leomilrib/junkie/blob/master/app.rb#L18) with the other _Octokit_ settings?